app/static/ringing_room.js

```diff
@@ -567,8 +570,13 @@ $(document).ready(function () {
 
             // a call was received from the server; display it and play audio
             make_call: function (call) {
-                this.display_message(call, 2000);
-                this.audio.play(call);
+                if (call.indexOf('sorry') != -1) {
+                    this.display_message(call, 2000);
+                    calls.play('SORRY');
+                } else {
+                    this.display_message(call_types[call], 2000);
```

Comment:
`call_types` is defined in `audio.js`; it's just a mapping object.

I had forgotten about command-line Wheatley, so I suppose I will need to do this now! My intention had been that if any unrecognized call (i.e. something not in `call_types`) is made, RR treats it as a `Change method` call for audio purposes but displays the original text. Does that still sound workable? Your `Go PB5` example makes me worried that we need to do something a bit more complex.

Reply:
@kneasle's looked a bit more at this, but change of method sounds right for everything that doesn't contain `Go ...`.
The `Go method` call from Wheatley would likely be in response to a human pressing `Go`, so it would be nice to have a solution to avoid duplicating the audio (`Go...Go`). The other case is up down in, which doesn't really need the audio for `Go ...` either.

At the moment Wheatley mostly just forwards calls from the Complib api, stripping out the call of `Stand`
e.g. https://api.complib.org/composition/77205/rows gives
- `Go Cheeky Little`
- `Cunecastre`
- `Bob`
- `That's all`

It was just a nice coincidence the capitalisation of calls matched between Complib and Ringing Room.

Reply:
I think this solution is good - it will break if someone splices to a method that starts with 'Go' but there isn't anything we can do about this really, since it is genuinely ambiguous.

Reply:
Ok, just to confirm: The preferred solution is:

1. If Wheatley sends a recognized call, do that.
2. If Wheatley sends a unrecognized call starting `Go...`, play no audio and display the original text
3. If Wheatley sends an otherwise-unrecognized call, play `Change method` and display the original text

Reply:
Yep I think that's the best thing we have.  We can't stop people being dumb and naming methods things like `Rounds`, but seriously if they do that then I can't say they shouldn't expect things to bug out.